overops-java-bullshifier/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for overops-java-bullshifier.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Overops Config settings
+overops:
+  collectorHost: ""
+  collectorPort: 6060
+  applicationName: ""
+  deploymentName: ""
+  serverName: ""
+  color: "white"
+  noAgent: false # Use for tests
+
+  # Use an agent-sidecard instead of an embedded v4 agent (https://github.com/takipi-field/kubernetes/tree/master/agent/sidecar).
+  agentSidecar:
+    enabled: false
+    useDebug: false
+    repository: overops/agent-sidecar
+    pullPolicy: IfNotPresent
+    tag: latest
+
+  # Job Related settings
+  job:
+    enabled: false
+    runnindDurationHours: 0
```

Comment:
Based on the Deployment and Job files looks like the "runnindDuration" values should be in the overops section not the job section. Also looks like a typo :) ? running instead of runnind

Reply:
The deployment get the default 0 minutes for runningMinutes .
The Job run 5 in as default